pages/t/[link].tsx

```diff
@@ -0,0 +1,50 @@
+
+//Get the links and filter to the corresponding link
+// Redirect to the actual link
+
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import React from 'react';
+import Layout from '../../components/Layout';
+import styles from '../../styles/Page.module.scss';
+import {linksQuery} from '../../utils';
+
+export default function LinkPage(): JSX.Element {
+  const router = useRouter();
+
+  if (typeof window !== 'undefined' && router.isFallback) {
+    void router.push('/');
+  }
+
+  return (
+    <Layout
+      content={(
+        <div id={styles.container}>
+          go back to home
+        </div>
+      )}
+    />
+  );
+}
+
+export const getServerSideProps: GetServerSideProps = async ({ params, res}) => {
+  const {link} = params;
+  const contentfulRes = await fetch(`https://graphql.contentful.com/content/v1/spaces/${process.env.SPACE_ID}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': `Bearer ${process.env.ACCESS_TOKEN}`,
+    },
+    body: JSON.stringify({query: linksQuery(link)}),
```

Comment:
this is very similar to the above case..

lets make function that takes in a query?